src/ContentTypeCache.ts

```diff
@@ -172,11 +172,15 @@ export default class ContentTypeCache {
         if (!machineNames || machineNames.length === 0) {
             return cache;
```

Comment:
Won't we need to make sure that the function is attached if all contentTypes are requested? (The return value you've changed below is only used in certain cases if you look for a specific contentType).

Reply:
You are completely right. My bad. I will change this today.